prov/verbs/src/fi_verbs.c

```diff
@@ -990,14 +1004,8 @@ fi_ibv_create_ep(const char *node, const char *service,
 
 	ret = rdma_create_ep(id, _rai, NULL, NULL);
 	if (ret) {
+		VERBS_INFO_ENO(FABRIC, "rdma_create_ep", errno);
 		ret = -errno;
-		if (ret == -ENOENT) {
-			FI_WARN(&fi_ibv_prov, FI_LOG_CORE,
-				"rdma_create_ep()-->ENOENT; "
-				"likely usnic bug, skipping verbs provider.\n");
-			ret = -FI_ENODATA;
-		}
-		FI_INFO(&fi_ibv_prov, FI_LOG_CORE, "Unable to create rdma_cm id\n");
```

Comment:
> The filtering logic wouldn't help but since in the active endpoint case, **the app passed a specific address to bind to**, which would be the address of a valid verbs device

This is the incorrect assumption (emphasis mine).  Consider the case of running `fi_msg_pingpong $DESTINATION_IP_ADDR`, where `$DESTINATION_IP_ADDR` is some valid IP address reachable from a usNIC device.  No source address is provided here.  The usNIC provider will contribute a proper result to the list returned by `fi_getinfo`, but the verbs provider will *also* try to contribute to the `fi_getinfo` result and will end up returning `ENOENT` for the whole `fi_getinfo` call instead of simply omitting verbs results from the returned list.

Anyway, don't worry about it, the PR is fine as-is.